hack/verify-flags/exceptions.txt

```diff
@@ -5,11 +5,13 @@ cluster/aws/templates/configure-vm-aws.sh:  # We set the hostname_override to th
 cluster/aws/templates/configure-vm-aws.sh:  api_servers: '${API_SERVERS}'
 cluster/aws/templates/configure-vm-aws.sh:  env-to-grains "hostname_override"
 cluster/aws/templates/configure-vm-aws.sh:  env-to-grains "runtime_config"
+cluster/aws/templates/configure-vm-aws.sh:  kubelet_api_servers: '${KUBELET_APISERVER}'
```

Comment:
the flag that you're introducing doesn't use underscores...

On Wed, Mar 2, 2016 at 12:52 PM, Hai Huang <notifications@github.com> wrote:

> In hack/verify-flags/exceptions.txt
> <https://github.com/kubernetes/kubernetes/pull/22246#discussion_r54761850>
> :
>
> > @@ -5,11 +5,13 @@ cluster/aws/templates/configure-vm-aws.sh:  # We set the hostname_override to th
> >  cluster/aws/templates/configure-vm-aws.sh:  api_servers: '${API_SERVERS}'
> >  cluster/aws/templates/configure-vm-aws.sh:  env-to-grains "hostname_override"
> >  cluster/aws/templates/configure-vm-aws.sh:  env-to-grains "runtime_config"
> > +cluster/aws/templates/configure-vm-aws.sh:  kubelet_api_servers: '${KUBELET_APISERVER}'
>
> Travis-ci built failed previously and suggested to run hack/verify-flags-underscore.py
> -e > hack/verify-flags/exceptions.txt. I think the reason is that the new
> flag we're introducing here "kubelet_api_servers" is also used by these
> scripts, but they're not related.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/22246/files#r54761850>.
>